diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter6/RunInScript.kt

```diff
@@ -44,36 +46,36 @@ class RunInScript(private val configRules: List<RulesConfig>) : Rule("run-script
     }
 
     private fun checkGradleNode(node: ASTNode) {
-        val astNode = if (node.firstChildNode.elementType == PARENTHESIZED) {
-            node.firstChildNode
+        val astNode = if (node.hasEqBinaryExpression()) {
```

Comment:
Please update kdocs or write some comments here, I can't remember how we are allowing dot-qualified expressions in scripts and there are no explicit checks.

Reply:
I updated KDoc and added dot-qualified expression in test.